packages/training/python/tests/integration/test_json_mode_integration.py

```
@@ -0,0 +1,554 @@
+            total_pnl=200.0,  # Better PnL
+        )
+        
+        inputs = TrajectoryRewardInputs(
+            final_pnl=10.0,
+            starting_balance=10000.0,
+            end_balance=10010.0,
+            format_score=0.7,
+            reasoning_score=0.7,
+        )
+        
+        high_social_score = archetype_composite_reward(inputs, "social-butterfly", high_social)
```

Ruff (0.14.10)

Local variable `high_social_score` is assigned to but never used

Remove assignment to unused variable `high_social_score`

(F841)